src/System.Private.Xml.Linq/tests/xNodeBuilder/ErrorConditions.cs

```diff
@@ -1181,6 +1181,14 @@ public void Variation42()
                     }
                     throw new TestException(TestResult.Failed, "");
                 }
+
+                private void CompareParamName(string actual, string expected, string message)
+                {
+                    if (PlatformDetection.IsNetNative) // ILC optimization sets ParamName always to null.
+                        return;
```

Comment:
The null happens only when built in ret mode. So if we build them in chk mode for debugging purposes or any other reason the ParamName wouldn't be null and they would all fail.